docs/supported_formats/image.rst

```diff
@@ -41,7 +41,7 @@ Extra saving arguments
   :py:func:`~.matplotlib.pyplot.imshow`.
 - ``**kwds`` : keyword arguments supported by the individual file
   writers as documented at
-  https://imageio.readthedocs.io/en/stable/formats.html when exporting
+  `imageio <https://imageio.readthedocs.io/en/stable/formats/index.html>`__ when exporting
```

Comment:
In case, the same links appear twice in the page, this is necessary to avoid sphinx warning. An alternative to use `__` instead of `_` may be to references but this one is easy!